evaluate.py

```diff
@@ -225,6 +234,7 @@ def main(args_not_parsed):
     fig_hist_generated = plt.figure()
     plt.bar(*np.unique(yhat.cpu(), return_counts=True))
     plt.title(f'generated {args.ckpt}')
+    plt.savefig('generated_bar_plot.png')
```

Comment:
in what path will the figure be saved? use a user-given path or refrain from saving.